src/firefly/java/edu/caltech/ipac/firefly/server/util/ipactable/JsonTableUtil.java

```diff
@@ -10,6 +10,7 @@
 import edu.caltech.ipac.util.DataGroup;
 import edu.caltech.ipac.util.DataType;
 import edu.caltech.ipac.util.StringUtils;
+import org.eclipse.jetty.util.StringUtil;
```

Comment:
Are you sure you are not using the StringUtils from Firefly instead?

Reply:
Good eyes.  It was inserted as by IntelliJ auto-complete feature.  Not used.  But, I'll remove it.